### ISSUE TITLE
Opção de ter acesso as antigas versões do po-ui na documentação

### ISSUE THREAD

**Issue by PedroHenrique1s** [po-ui/po-angular#2222]
#### Descrição da melhoria
Deveria ter uma opção de alterar a versão que está usando da Documentação do PO-UI, para realizar consultas de componentes, ícones além da própria documentação dos componentes da versão que está desenvolvendo

#### Caso de uso
Exemplo nessa nova versão foi feito as mudanças dos icones porém a versão que utilizo o po-ui utiliza a antiga norma de ícones, servindo isso pros componentes da documentação também ficando mais díficil ainda de como funciona o tipo do componente

**Comment by renan-augusto** [comment]
Seria interessantíssimo termos esse acesso hoje, acredito que só deixaria a biblioteca mais completa ainda!

**Comment by MatheusBugatti** [comment]
Concordo com o que foi dito, a documentação de versões são muito importante na hora do desenvolvimento. Seria muito interesse que pudéssemos alterar para a versão desejada.

**Comment by PedCoelho** [comment]
Concordo em gênero, número e grau com a necessidade levantada pelo @PedroHenrique1s

**Comment by camposmichel** [comment]
Up 🔝

**Comment by wagnerras** [comment]
Concordo plenamente. Faz muita falta.

**Comment by tiagogomesdeveloper** [comment]
Up 🔝
E poderiam deixar disponível em alguma parte da documentação a **descrição e imagem** dos ícones antigos para facilitar a implementação dos projetos que ainda utilizam a versão anterior.
